src/datasets/other/addons_daily/reference.md

```diff
@@ -3,17 +3,122 @@
+## Introduction:
 
-This part of the project should allow someone to learn everything there is to know about extension X after looking in one place (at this point the “place” is a dataset). This repo accomplishes all of the above, the ideal next steps involve sharing these insights with a broader audience.
+The addons_daily dataset serves as the central hub for all Firefox extension related questions. This includes questions regarding browser performance, user engagement, click through rates, etc. Each row in the table represents a unique addon, and each column is a unique metric.
 
-### Sharing Insights 
-Developers get a narrow view of their extension in the wild through AMO’s dashboards (i.e. see Privacy Badger’s Dashboard). These dashboards are great for tracking user uptake and and language/OS breakdowns, however it completely omits user behavior. 
+### Contents
+Prior to construction of this dataset, extension related data lived in several different sources. Addons_daily has combined metrics aggregated from several sources, including raw pings, telemetry data, and google analytics data.
 
-For instance, how long does it take extension X’s browser action (the button on the Firefox toolbar) to load compared to the average? How many tabs do users with extension X typically use compared to the average? Compared to extension Y? These are just some examples of things a developer would interested to know when prioritizing improvements to their software (and also very useful for Mozilla’s internal use!).
+### Accessing the Data
+The data is stored as a parquet table in S3 at the following address:
 
-This part of the project should first allow Mozilla to better understand how individual extensions are behaving with minimal effort by way of a dashboard/report. Lastly, the hope is to share a subset of this data with developers/the public. An idea what this might look like is a lightweight version of the public data report sans any commentary (since there will be hundreds of add-ons), or a more zoomed out representation of TMO.
+s3://net-mozaws-prod-us-west-2-pipeline-analysis/bmiroglio/addons_daily_test/
 
+***The addons_daily table is accessible through re:dash using the Athena data source. It is also available via the Presto data source, though Athena should be preferred for performance and stability reasons.***
+
+## Data Reference
+
+### Example Queries
+
+
+#### Query 1
+
+
+```
+SELECT addon_id,
```

Comment:
For the example queries, write a sentence explaining the interpretation of the query. Since the data is in re:dash now, you should link to a valid re:dash query as well.